### PR TITLE
[stable17] Use handlebars from node_modules (on CI)

### PR DESCRIPTION
**Pull request by skjnldsv**

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

**Comment by rullzer** [comment]
https://drone.nextcloud.com/nextcloud/server/21998/4/3 ci says no

File: .drone.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@ steps:
 - name: handlebars
   image: node
   commands:
-    - npm install handlebars -g
+    - npm ci
```
Comment (ChristophWurst) [inline .drone.yml]: :-1: actually, please don't modify backported code unless this is really necessary. This does not match the changes from https://github.com/nextcloud/server/pull/17340/files

Comment (skjnldsv) [reply .drone.yml]: Yep, but let's wait for tests!
Because it works on others, so I'm pretty sure we'll have to backport ti to master!

**Comment by ChristophWurst** [comment]
> https://drone.nextcloud.com/nextcloud/server/21998/4/3 ci says no

@ChristophWurst says no. This build is from another PR. The original commit actually passed the Handlebars check :)

File: .drone.yml (hunk 1 above)
Comment (ChristophWurst) [reply .drone.yml]: See https://drone.nextcloud.com/nextcloud/server/22033. It passed already.

Comment (skjnldsv) [reply .drone.yml]: https://drone.nextcloud.com/nextcloud/server/22035/4/3
So, I was wrong!

Why are webpack build tests passing though? :O

**Comment by skjnldsv** [comment]
tests passes

**Comment by skjnldsv** [comment]
Reset back to first backport

**Review by ChristophWurst — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Pull request merged**